src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/intervener/IntervenersAboutToSubmitHandler.java

```diff
@@ -28,6 +31,8 @@
 @Service
 public class IntervenersAboutToSubmitHandler extends FinremCallbackHandler {
     private final IntervenerService service;
+    private static final List<String> ADD_OPERATION_CODES = Arrays.asList(ADD_INTERVENER_ONE_CODE, ADD_INTERVENER_TWO_CODE, ADD_INTERVENER_THREE_CODE, ADD_INTERVENER_FOUR_CODE);
```

Comment:
Use List.of rather than Arrays.asList. The latter returns a modifiable list

Reply:
[1a9514c](https://github.com/hmcts/finrem-case-orchestration-service/pull/1641/commits/1a9514c2a51c04312192384722a4f1d21a0316d7)